prow/tide/tide.go

```diff
@@ -35,13 +37,20 @@ import (
 	"k8s.io/test-infra/prow/pjutil"
 )
 
+const (
+	statusContext   string = "tide"
```

Comment:
Capitalize the status context name?

Reply:
It seems like the standard is to not capitalize this. Travis' is `continuous-integration/travis-ci`, we have `cla/linuxfoundation`, and so on.